doc/support_tasks/create_new_report.md

```diff
@@ -0,0 +1,84 @@
+# Creating a new report
+Very rarely a new Delivery partner organisation may join BEIS ODA reporting.
+
+When this happens a developer will be required to 'seed' the new reports into
+the database for the new organisation.
+
+Once the seed is planted, no further developer action is required as all
+subsequent reports are created automatically.
+
+This request should come through as a support ticket and will need to clearly
+indicate the new delivery partner organisation that must have been created by
+BEIS already and which fund/s to create the report for, at the time of
+writing there are only two options: Newton fund or Global Challenges Research
+Fund (GCRF).
+
+## Process
+
+### Prerequisites
+- [Console access](../console-access.md)
+- The Delivery partner organisation
+- The funds to create the reports for
+
+### Steps
+Get access to the Rails console in production, see [console
+access](../console-access.md)
+
+Get the Delivery partner organisation id, one approach is to list all the
+delivery partner organisations and find the id of the one in question:
+
+```
+Organisation.delviery_partner
+
+organisation = Organisation.find("{id}")
+```
+
+Get the id of the funds, again one approach is to list all fund activities and
+assign the ids:
+
+```
+Activity.fund
+
+gcrf_fund = Activity.find("{gcrf_fd}")
+newton_fund = Activity.find("{newton_id}"
+```
```

Comment:
Might be easier to use the Fund objects here, ie: 

```
Fund.by_short_name("NF").activity.id
```

for Newton

and:

```
Fund.by_short_name("GCRF").activity.id
```

for GCRF